mediapipe/calculators/instantmotiontracking/sticker.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Header file serving as blueprint and implementation for a sticker object
+
+class Sticker {
+  public:
+    float xAnchor, yAnchor, // X,Y anchors (updated by tracking subgraph)
+
+    userRotation, userScaling; // rotation from user in RADIANS, and scaling from user in PIXELS
+
+    int renderID, stickerID; // ID of asset to render, ID of individual sticker object
```

Comment:
C++ code style for naming is "render_id", "sticker_id", different from java code style.

Also prefer one variable per line.